templates/base/README.md

```diff
@@ -114,6 +114,14 @@ You can verify your smart contract on Etherscan by running:
 yarn verify --network network_name
 ```
 
+eg: `yarn verify --network sepolia`
+
+This uses [etherscan-verify from hardhat-deploy](https://www.npmjs.com/package/hardhat-deploy#4-hardhat-etherscan-verify) to verify all the deployed contracts.
+
+You can alternatively use [hardhat-verify](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify) to verify your contracts by running `yarn hardhat-verify --network network_name contract_address`
```

Comment:
There should be also be constructor arguments see https://arc.net/l/quote/fqupvdti

Reply:
Ohh thanks Rinat! update at [71be8e7](https://github.com/scaffold-eth/scaffold-eth-2/pull/685/commits/71be8e7379b10b240d06ca7e7393ff7e2a856505) ....Also just learned about this Arc browser quote link feature love it !! :)

Thanks 🙌

Reply:
Great, thanks!